src/core/network.js

```diff
@@ -75,6 +75,24 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if (FIREFOX || MOZCENTRAL)
+//var supportsMozChunked = true;
+//#endif
+//#if GENERIC
```

Comment:
Nit: Since there is a `SINGLE_FILE` build target as well, would it perhaps make more sense to just use `//#else` instead of
```javascript
//#endif
//#if GENERIC
```